package.json

```diff
@@ -220,7 +220,8 @@
         "AppImage"
       ],
       "desktop": {
-        "MimeType": "application/epub+zip"
+        "MimeType": "application/epub+zip",
+        "Keywords": "Audiobook;Book;Ebook;EPUB;Reader;Readium;Viewer;"
```

Comment:
@llemeurfr any other "keyword" suggestions?

Syntax specification:
https://developer.gnome.org/documentation/guidelines/maintainer/integrating.html#desktop-files

Electron Builder doc (Linux-specific section):
https://www.electron.build/configuration/linux
=>
https://github.com/edrlab/thorium-reader/blob/2ad37fa8d4753c308b6d39b3f7c9439e09d93447/package.json#L217-L227

...Windows:
https://www.electron.build/configuration/win
=>
https://github.com/edrlab/thorium-reader/blob/2ad37fa8d4753c308b6d39b3f7c9439e09d93447/package.json#L210-L216

...and Mac:
https://www.electron.build/configuration/mac
=>
https://github.com/edrlab/thorium-reader/blob/2ad37fa8d4753c308b6d39b3f7c9439e09d93447/package.json#L228-L236


Also note the general `fileAssociations` field (keywords there too, but for a different purpose):

https://github.com/edrlab/thorium-reader/blob/2ad37fa8d4753c308b6d39b3f7c9439e09d93447/package.json#L163-L209